R/decontam.R

```diff
@@ -101,21 +149,21 @@ isContaminant <- function(seqtab, conc=NULL, neg=NULL,
   if(normalize) seqtab <- sweep(seqtab, 1, rowSums(seqtab), "/")
   method <- match.arg(method)
   if(method == "auto") {
-    if(!missing(conc) && missing(neg)) method <- "frequency"
-    else if(missing(conc) && !missing(neg)) method <- "prevalence"
+    if(!is.null(conc) && is.null(neg)) method <- "frequency"
```

Comment:
Could you describe the reason for these changes from using `missing` to using `is.null` in parsing whether or not certain arguments were provided to the functions?

Reply:
In the previous version, all arguments had default values, but you were testing or had to test for missing. When I converted it to S4 dispatch some of the test didn't trigger anymore causing problem downstream.